js/sykmeldinger/sykmelding-avvist/SykmeldingDiagnoser.js

```diff
@@ -0,0 +1,93 @@
+/* eslint arrow-body-style: ["error", "as-needed"] */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { getLedetekst } from '@navikt/digisyfo-npm';
+import Hjelpetekst from 'nav-frontend-hjelpetekst';
+import { smDiagnosePt } from '../../propTypes/smSykmeldingProptypes';
+import { Nokkelopplysning } from './Nokkelopplysning';
+
+
+export const Hoveddiagnose = ({ vis = true, hoveddiagnose = null }) => {
```

Comment:
Har du vurdert å sende inn hele medisinskVurdering til disse to diagnosekomponentene? Da blir det mindre å sjekke på i komponenten som bruker dem. Vet ikke om det blir bedre? 🤷‍♂

Reply:
Ja... men de skal ha samme logikk for visning egentlig... så jeg ville ha den utenfor. Vet ikke om det er det lureste da 😅